client/src/main/java/org/apache/rocketmq/client/impl/consumer/DefaultMQPullConsumerImpl.java

```diff
@@ -601,16 +601,15 @@ public synchronized void start() throws MQClientException {
     }
 
     private void checkConfig() throws MQClientException {
-        // check consumerGroup
-        Validators.checkGroup(this.defaultMQPullConsumer.getConsumerGroup());
-
         // consumerGroup
-        if (null == this.defaultMQPullConsumer.getConsumerGroup()) {
+        if (UtilAll.isBlank(this.defaultMQPullConsumer.getConsumerGroup())) {
```

Comment:
Maybe we should keep isBlank() check in checkGroup() and add FAQUrl.suggestTodo, and remove the null check after checkGroup(), in order to reduce duplicated code.